pkg/systemd/quadlet/quadlet.go

```diff
@@ -1800,6 +1803,17 @@ func getAbsolutePath(quadletUnitFile *parser.UnitFile, filePath string) (string,
 	return filePath, nil
 }
 
+func handleExitPolicy(unitFile *parser.UnitFile, groupName string, podman *PodmanCmdline) {
+	exitPolicy, found := unitFile.Lookup(groupName, KeyExitPolicy)
+
+	podman.add("--exit-policy")
+	if found {
+		podman.add(exitPolicy)
+	} else {
```

Comment:
no biggie but if one of these options is kind of a default, we will often set that and use a single condition as an override.  maybe consider fix if you repush with someone else's comment (only if you consider one of them to be the default).